config/initializers/omniauth.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  OmniAuth.config.allowed_request_methods = [:post]
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'],
+    {
+      scope: 'userinfo.email',
+      prompt: 'select_account'
+    }
+end
```

Comment:
This is sort of the Flux Capacitor of the Google OAuth login method.  Line 2 originally also allowed `:get`, but RSpec kept telling me that made the app unsecure, so out it went!